File: ConfiguratorService/app/Utils/Logger.py
import logging

class Logger:
    def __init__(self, log_file_path='app.log'):
        logging.basicConfig(filename=log_file_path, level=logging.INFO)
        self.logger = logging.getLogger(__name__)

    def log_action(self, action_description):
        self.logger.info(action_description)

File: API/API/EntityFramework/Utils/Logger.cs
using Serilog;

namespace Userdata.Utils
{

    public  class Logger
    {
        private readonly Microsoft.Extensions.Logging.ILogger logger;

        public Logger(string logFilePath = "app.log")
        {
            Log.Logger = new LoggerConfiguration()
                .WriteTo.File(logFilePath)
                .CreateLogger();

            var factory = LoggerFactory.Create(builder =>
            {
                builder.AddSerilog();
            });

            this.logger = factory.CreateLogger<Logger>();
        }

        public void LogAction(string actionDescription)
        {
            this.logger.LogInformation(actionDescription);
        }
    }
}
 
